docs/hopr-documentation/docs/developers/connecting-node.md

```diff
@@ -0,0 +1,135 @@
+---
+id: connecting-node
+title: Interacting with a HOPR node
+---
+
+Once you have your HOPR cluster up and running, you are ready to connect to a single HOPR node and listen to messages sent to it by other HOPR nodes.
+The following guide will show you how to connect to a running HOPR node, and verify some basic functionality using both the REST and WebSocket
+endpoints.
+
+## Connect to a HOPR node
+
+### 1. Get the node security credentials
+
+To avoid unsecured access to your HOPR node, all HOPR node's WebSocket and REST calls require an `apiToken`. Your API token needs to be appended as a
+`query` parameter for WebSocket connections and as encoded token via the `Authorization` header for you to be able to connect to it.
+
+:::info Tip
+The flag used to set this value in a HOPR node via `hoprd` is `--apiToken`. The default `apiToken` used across our documentation is the following one 
+(click on the right to copy), so make sure to change it when running your own node in a public network.
+<br/>
+
+```
+^^LOCAL-testing-123^^
+```
+:::
+
+Export your `apiToken` to be used in the incoming commands.
+
+```
+export apiToken="^^LOCAL-testing-123^^"
+```
+
+### 2. Test REST API connectivity
+
+Given your node's `apiToken` and your `HOPR_NODE_1_HTTP_URL` from the last section, try to send a REST command to query it’s address with the following `curl`
```

Comment:
Minor thing: I would include a line saying e.g. `HOPR_NODE1_HTTP_URL=localhost:3001` just for clarity and reminder. Also because of the port number that needs to be included in the URL and might not be immediately obvious.

Reply:
```suggestion
Given your node's `apiToken` and your `HOPR_NODE_1_HTTP_URL` from the last section (likely `localhost:3001`), try to send a REST command to query its address with the following `curl`
```

Good stuff, let me use the suggestion feature and do the change directly.